sdk/src/test/java/io/radar/sdk/RadarTest.kt

```diff
@@ -32,6 +33,16 @@ class RadarTest {
         private val apiHelperMock = RadarApiHelperMock()
         private val locationClientMock = RadarMockLocationProvider()
         private val permissionsHelperMock = RadarPermissionsHelperMock()
+
+        @BeforeClass
+        @JvmStatic
+        fun setUp() {
+            Radar.initialize(context, publishableKey)
+
+            Radar.apiClient.apiHelper = apiHelperMock
+            Radar.locationManager.locationClient = locationClientMock
+            Radar.locationManager.permissionsHelper = permissionsHelperMock
+        }
```

Comment:
I think we have some conflicts here since we did not rebase off your main development branch. Let's go and merge the other one and then rebase here